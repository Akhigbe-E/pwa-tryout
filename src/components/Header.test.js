import React from "react";

import { shallow } from "enzyme";
import { Header } from "./Header";

it("test screenshot of Header component", () => {
  expect(shallow(<Header />)).toMatchSnapshot();
});
