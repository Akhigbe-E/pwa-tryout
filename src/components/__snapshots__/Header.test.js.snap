// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test screenshot of Header component 1`] = `ShallowWrapper {}`;
